azerothcore/azerothcore-wotlk
# fix(DB/Event): Attempt to resolve STV Fishing Extravagnaza spawns and…

… start





## Changes Proposed:
-  Correct Holiday
-  Adjust start and end times
- All values are now equal to TC
- Update turn-in creatures to spawn with the turn-in event


## Issues Addressed:
- Closes https://github.com/azerothcore/azerothcore-wotlk/issues/4066



## Tests Performed:
- None



## How to Test the Changes:

See so that the events start at the correct time

as for npc, just do `.event start 62` and they will spawn


## Target Branch(es):








 
## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here in the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR
